src/pages/logos.tsx

```diff
@@ -129,34 +184,46 @@ export default function Logos({ exhibitorData }: LogosProps) {
   const startupPackages = exhibitorData.filter(
     (e) => e.package === "startup" && e.logoColor
   );
-
+  const baseAndStartUpPackages = [...basePackages, ...startupPackages];
+  const sponsorAndHeadhunterAndPremiumPackages = [
+    ...headhunterAndPremiumPackages,
+    ...sponsorPackages,
```

Comment:
Do we want to have these together? Quite sure that headhunter and premium are more expensive than sponsor so they should probably be treated differently on this page